blocks/cards/cards.js

```diff
@@ -35,9 +35,12 @@ export default async function decorate(block) {
   /* change to ul, li */
   const ul = document.createElement('ul');
   [...block.children].forEach((row) => {
+    // wrap main card element in an anchor
     const li = document.createElement('li');
-    li.innerHTML = row.innerHTML;
-    [...li.children].forEach((div) => {
+    li.innerHTML = '<a class="test-a">link</a>';
+
+    li.querySelector('a').innerHTML = row.innerHTML;
```

Comment:
For someone coming in cold to this, I fear this could be confusing. Let me take a stab at it.

Reply:
Definitely, as mentioned in a post above:

> @fkakatie, opening the discussion for [7ffbebd](https://github.com/hlxsites/pga-presidents-cup/pull/3/commits/7ffbebd5ac6793aeb977bb13fbfb53dbdbd9bc02):
it's a draft for now, trying to address the non explicit enough a text in the cards (Tickets, Hospitality, History). My idea is to wrap the whole card in an a so for SEO (and accessibility?) the text of it will the whole text in the card so Card Title + LEARN MORE) and transform the "LEARN MORE" a in a button.
Technically it works but it's certainly not the cleanest. I wonder though if a cleaner approach does not imply to change the block structure in the documents.
Also, it makes the whole card clickable, is it an issue?

This is more a something purely technical that would do the job but does not look appropriate. Thinking a bit more about this, it would be simpler to change the document itself, replace the "LEARN MORE" button by a respective TICKETS, HOSPITALITY and HISTORY button. Though, of course, this would need to be validated by the customer.

Reply:
I really, _really_ don't want to get in the business of making code assumptions (especially with hard-coded strings) for something that can and should be handled on the author side. If it's a matter of making the text in the titles and buttons more descriptive, that is definitely feedback we should give to the authors to do with what they will. Making the whole card clickable is non-issue, but the `test-a` class and that same `a` being query-selected three times was jarring.